src/features/left-sidebar/LeftSidebar.js

```diff
@@ -78,6 +80,8 @@ type MenuItem = {
     menuItems?: Array<SubMenuItem>,
     /** Localized text string to use for individual menu items */
     message: string,
+    /** Optional left side bar link renderer. Defaults to defaultNavLinkRenderer */
+    navLinkRenderer?: Function,
```

Comment:
can you change Function to actual function signature () => React.Node or whatever it might be, here and above